addons/Pouches/Pouches.lua

```diff
@@ -73,9 +86,13 @@ windower.register_event('addon command', function(...)
             end
         end
         if item.count > 0 then
-            log('Found '..item.count..' '..item.name..'. Commencing Use.')
-            log('You may simply type /heal to stop.')
-            use_item()
+            if inventory_info.current < inventory_info.max then 
+                log('Found '..item.count..' '..item.name..'. Commencing Use.')
+                log('You may simply type /heal to stop.')
+                use_item()
+            else
+                log("Your inventory is full...Aboriting")
```

Comment:
Single quotes and `Aboriting` again :D Also add a space after the ellipsis, or just use a comma (or a full stop).